org.openhab.ui.habpanel/pom.xml

```diff
@@ -15,116 +15,17 @@
 
     <packaging>eclipse-plugin</packaging>
 
-    <properties>
-        <ohc.version>2.5.0-SNAPSHOT</ohc.version>
-    </properties>
-
-    <scm>
-        <connection>scm:git:${scm.gitBaseUrl}/org.openhab.ui.habpanel.git</connection>
-        <developerConnection>scm:git:${scm.gitBaseUrl}/org.openhab.ui.habpanel.git</developerConnection>
-        <url>https://github.com/openhab/org.openhab.ui.habpanel</url>
-        <tag>HEAD</tag>
-    </scm>
-
-    <distributionManagement>
-        <repository>
-            <id>bintray</id>
-            <url>https://api.bintray.com/maven/openhab/mvn/org.openhab.ui.habpanel/;publish=1</url>
-        </repository>
-        <snapshotRepository>
-            <id>jfrog</id>
-            <url>https://openhab.jfrog.io/openhab/libs-snapshot-local</url>
-        </snapshotRepository>
-    </distributionManagement>
-
     <dependencies>
         <dependency>
             <groupId>org.openhab.core.bundles</groupId>
             <artifactId>org.openhab.core</artifactId>
             <version>${ohc.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.openhab.core.bundles</groupId>
-            <artifactId>org.openhab.core.ui.dashboard</artifactId>
-            <version>${ohc.version}</version>
+            <groupId>org.openhab.ui</groupId>
```

Comment:
Do you really need to cause merge problems with my PR 😛

Reply:
I think it's just POM changes that may conflict. If your changes are better just replace the whole POM with yours. :-)